packages/trader/src/Modules/CFD/Components/cfd-poa.jsx

```diff
@@ -51,7 +51,7 @@ const CFDPOA = ({ onSave, onCancel, index, onSubmit, refreshNotifications, ...pr
 
         const validations = {
             address_line_1: [v => !!v && !v.match(/^\s*$/), v => validAddress(v), v => validLength(v, { max: 70 })],
-            address_line_2: [v => !v || validAddress(v), v => validLength(v, { max: 70 })],
+            address_line_2: [v => !v || validAddress(v), v => !v || validLength(v, { max: 70 })],
```

Comment:
```
export const validLength = (value = '', options) =>
``` 
Can we instead default value param to empty string? This is to improve code readability.